doc/neomutt.man

```diff
@@ -209,7 +209,8 @@ The help line for the index menu is not updated if you change the bindings
 for one of the functions listed while Neomutt is running.
 .PP
 For a more up-to-date list of bugs, errm, fleas, please visit the
-neomutt project's bug tracking system under http://bugs.neomutt.org/.
+neomutt project's bug tracking system under https://github.com/neomutt/neomutt/issues.
+Keep in mind that some very hard to solve bugs are closed for now.
```

Comment:
I don't think the last sentence is adding a lot of information and has a bit of a negative ring to it. We should encourage user contributions even in the form of new issues, and the qualifier "some very hard to solve bugs" is nebulous at best. What's "very hard to solve" is something that needs to be determined on a case by case basis and hopefully commented on in the the issue itself. I would prefer if the the sentence was removed.

Reply:
Hm. Well. @flatcap likes to keep the list of issues short and "solvable". That means, relevant issues are closed and maybe reopened some day. That means, i want to make sure users don't think of the open bugs as "all known issues", as that leaves the hard to solve out.

Keeping that in mind, do you have a better idea for expressing that?

Reply:
@toogley: You could link both, https://github.com/neomutt/neomutt/issues and https://github.com/neomutt/neomutt/issues?q=is%3Aissue+label%3Adifficulty%3Ahard.

Once https://github.com/isaacs/github/issues/660 is addressed, one could go for sth like https://github.com/neomutt/neomutt/issues?q=is%3Aissue+is%3Aopen+OR+label%3Adifficulty%3Ahard.

For clarity, maybe a new label (wontfix/postponed/?) would be even more clear than just 'difficulty:hard'.

Reply:
@mschilli87 Hm, thanks :) So, @flatcap I'd like to have your opinion here.